app/processing/reverse-engineer-invoice-number.js

```diff
@@ -0,0 +1,5 @@
+const reverseEngineerInvoiceNumber = (invoiceNumber) => {
+  return `original${invoiceNumber.slice(0, 5)}`
```

Comment:
I know this is just a placeholder so this isn't a suggestion for change, more just a snippet of wisdom.

Because we're subscribing to all submitted payments, we're going to pass Vets, Lump Sums, Pilot and SFI through this.  All the Siti Agri ones we'll be able to reverse engineer back to the SITI invoice to match to DWH but with slightly different values for what you have as `original`.  Vets doesn't have an original invoice number as they don't provide us with one.  We create it for them.